<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gomoku</class>
 <widget class="QMainWindow" name="Gomoku">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gomoku</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>160</width>
      <height>201</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="createButton">
       <property name="text">
        <string>创建</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="undoButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>悔棋</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="tipButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>提示</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>认输</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>读取</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="GameBoard" name="gameboard" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>460</width>
      <height>460</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>460</width>
      <height>460</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>151</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="colorLabel">
       <property name="styleSheet">
        <string notr="true">font: 18pt &quot;.Helvetica Neue DeskInterface&quot;;
background-color: rgb(170, 89, 99);</string>
       </property>
       <property name="lineWidth">
        <number>3</number>
       </property>
       <property name="text">
        <string>等待连接</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="countDown">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(76, 170, 83);</string>
       </property>
       <property name="value" stdset="0">
        <double>60.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>敌方用时</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="enemyTimeCount">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 0, 0);</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>我方用时</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="playerTimeCount">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(0, 0, 255);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GameBoard</class>
   <extends>QWidget</extends>
   <header>gameboard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
